onboarding.md

```diff
@@ -1,279 +1,289 @@
+It should be used, for example, to note vacations, conference travel, and other workday conflicts.
+All other events should go in the Core Events calendar.
+In general, this calendar is for events that could possibly involve 3 or more lab members.
+Mandatory events such as lab meetings, scrums, and group deadlines go on Core Events.
+
+**Accounts:** Lab members are expected to have accounts for the following and be members of the specified (organizations) if applicable:
+
++ GitHub (greenelab)
++ Google Calender (Shared Calendar)
++ Slack (GreeneLab)
+
+## Meetings
+
+**Scrum:** Our team's scrum process involves three components:
+
+    1. An opening scrum meeting at 9:45 AM Monday morning where individuals will lay out their goals for the week, in the form of post-it notes, on the whiteboard in the Greene Lab.
+    2. A closing scrum meeting at 1:40 PM Friday afternoon where team members will reflect on the progress they made on the goals they stated the previous Monday.
+    3. A daily virtual scrum update.
+
+The daily virtual scrum update should include an update to [scrum repository](https://github.com/greenelab/scrum).
+An issue is automatically created for each day the office is open.
+These issues can be found [here](https://github.com/greenelab/scrum/issues).
+The update should include the following:
+
+    1. What specific item(s) he/she accomplished yesterday.
+    2. What specific item(s) he/she plans to accomplish today.
+    3. Who, if anyone, is blocking him/her?
+    4. Who, if anyone, is he/she blocking?
+
+**Lab Meeting:** Lab meeting is held weekly at a location at Penn and also virtually via [uberconference](https://uberconference.com/greenescientist).
+Scheduling is managed via a google spreadsheet.
+See the `#general` slack channel's pinned items link.
+Lab meeting consists of two components described below.
+
+- Journal Club
+- Braintrust
+
+**Journal Club:** This is an opportunity for lab members to strengthen their ability to comprehend and critically analyze research that has already been conducted.
+For journal club, everyone will be randomly split off into groups of two (or three if an odd number of lab members).
+Presenters in these groups are expected to present on four papers.
+Three papers will be in the form of lightning talks - where presenters spend 5 to 10 minutes discussing the following: the overall goal of the work, any results presenters found interesting and the paper's implication for presenter's own research or any potential implications that touch on other ongoing research in the lab.
+The final paper will be a more in-depth discussion where presenters will dive deeper into individual experiments and mention how these experiments tie into the paper's global scheme. 
+All selected papers except for one should have a publication date that is later than the most recent paper from your last journal club presentation.

```

Comment:
For the first presentation, the paper date from the last presentation is NaN :D
Should we say published in the last year?

Reply:
>For the first presentation, the paper date from the last presentation is NaN :D
Should we say published in the last year? 

No. I'll update the section to make it clear for new presenters.